nixos/modules/installer/cd-dvd/installation-cd-graphical-plasma5.nix

```diff
@@ -1,5 +1,5 @@
 # This module defines a NixOS installation CD that contains X11 and
-# Plasma5.
+# KDE 5.
```

Comment:
"Plasma 5" or "KDE Plasma 5" is the correct name, there is no such thing as KDE 5.

Reply:
Yeah that got there by accident